File: core/remove/removeChar/removeChar.py
def remove_char(s):
    return s[1:-1]

    # return s[slice(1, -1)]

# 
def remove_char(s):
    if len(s) > 2:
        return s[1:-1]
    return ''  # Return an empty string if the input is too short

# 
import re
def remove_char(s):
    return re.sub(r'^.(.*).$', r'\1', s)
# 

def remove_char(s):
    lst = list(s)
    return ''.join(lst[1:-1])

File: core/remove/removeChar/removeChar.js
// function that removes the first and last characters of a string.

function removeChar(str) {
  // return str.substring(1, str.length - 1);
  // return str.substr(1, str.length - 2);

  return str.slice(1, str.length - 1);
  // str.slice(1,-1)  // argument -1 is the same as 'str.length -1'
}

function removeChar(str) {
  const [, ...rest] = str.split("");
  rest.pop();
  return rest.join("");
}

function removeChar(str) {
  const match = str.match(/^.(.*).$/);
  return match ? match[1] : "";
}

function removeChar(str) {
  return [...str].slice(1, -1).join("");
}
